### PR TITLE
build: input component

### PR DESCRIPTION
**Pull request by Shubhamkr2610**

### CODE REVIEW

File: components/input/input.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            </iframe>
+            <br>
+            <h2 class="input-heading">Form Input</h2>
+            <br>
+            <div class="textarea-container">
+                <form class="form">
+                    <input class="form-input" id="primary-input" type="email" name="email-id"
+                        placeholder="Enter Your Email-Id">
+                    <input class="form-input" type="password" name="password" placeholder="Enter Your Password">
+                    <button class="primary-button error-button">login</button>
```
Comment (DivyanshSareen) [inline components/input/input.html:100]: everything looks good, just make sure adding input labels won't break any layout. 

would recommend this article to know more about labels
https://css-tricks.com/html-inputs-and-labels-a-love-story/

**Review by DivyanshSareen — APPROVED** [review]

File: components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    flex-direction: column;
+    border: 1px solid var(--alert-secondary-color);
+    padding-top: 1rem;
+    margin-left: 5rem ;
+    margin-bottom: 2rem;
+    padding-left: 10rem;
+    box-shadow: 0px 3px 5px var(--alert-secondary-color);
+}
+.textarea{
+
```
Comment (zoef-yt) [inline components/input/input.css]: remove extra spacing

File: components/input/input.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    resize: none;
+}
+.primary-button{
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    padding: 10px;
+    width: 5rem;
+    font-size: 1rem;
+    background-color: var(--primary-color);
+    color: white;
```
Comment (zoef-yt) [inline components/input/input.css]: use var for colour

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: components/input/input.css (hunk 2 above)
Comment (Shubhamkr2610) [reply components/input/input.css]: fixed . PLease review

File: components/input/input.css (hunk 3 above)
Comment (Shubhamkr2610) [reply components/input/input.css]: done. please review

**Review by zoef-yt — APPROVED** [review]

**Pull request merged**